redbot/cogs/economy/economy.py

```diff
@@ -247,6 +247,45 @@ def __init__(self, bot: Red):
                 )
             )
 
+    @_bank.command()
+    @commands.guild_only()
+    @checks.guildowner()
+    async def removedead(self, ctx, confirmation: bool = False):
```

Comment:
I don't love this command name as it isn't very clear what this will do (it is unclear what "dead" means).

Reply:
Read below: https://github.com/Cog-Creators/Red-DiscordBot/pull/2845#discussion_r301943785